scripts/run-tests.sh

```diff
@@ -28,6 +28,7 @@ set -ex
 # FIXME: make helm lint work
 # helm lint charts/renku -f charts/minikube-values.yaml
 
```

Comment:
hmm, ok - slowly passing builds are better than quickly failing ones I gues...

Reply:
The problem is that when the chart ends, gitlab is not fully ready. The rest api endpoints sometime take a little bit longer to be up.

Reply:
so maybe the readiness probe should hit `/api/v4` instead?